fatrace.c

```diff
@@ -251,6 +253,12 @@ print_event (const struct fanotify_event_metadata *data,
                 len--;
             procname[len] = '\0';
             procname_pid = data->pid;
+
+            /* get user and group */
+            if (option_user == 1) {
```

Comment:
Please treat this as a boolean option, i.e. `if (option_user)`. In other words, giving it twice with `-uu` should still show users.

Reply:
Done.